ketcher-autotests/tests/utils/canvas/helpers.ts

```diff
@@ -29,8 +29,9 @@ export async function drawElementByTitle(
 }
 
 export async function getLeftToolBarWidth(page: Page): Promise<number> {
-  const leftBar = await page.locator('[class^="LeftToolbar-module_root"]');
-  const leftBarSize = await leftBar.boundingBox();
+  const leftBarSize = await page
+    .getByTestId('LeftToolbar-module_root')
```

Comment:
I think this kind of classes was generated by some library and my proposal is to have more meaningful names. May be something like `.getByTestId('left-toolbar')`

Reply:
I agree with this comment